ic-stable-structures/src/storage/structures/vec.rs

```diff
@@ -0,0 +1,182 @@
+use std::collections::BTreeMap;
+
+use ic_exports::{
+    ic_kit::ic,
+    stable_structures::{memory_manager::MemoryId, BoundedStorable, Vec},
+    Principal,
+};
```

Comment:
In our code style guide we decided to split such imports on module basis:
https://infinityswap.atlassian.net/wiki/spaces/CPROD/pages/24444929/Rust+Conventions#Import--/-use

Reply:
Thanks, fixed. For some reason `cargo +nightly fmt` didn't change that